splink/linker.py

```diff
@@ -1004,9 +921,7 @@ def _predict_warning(self):
             logger.warning(warn_message)
 
     def _table_exists_in_database(self, table_name):
```

Comment:
yep exactly - I just thought for this first step I would keep these aliases in place so that we didn't need to adjust every call site, which can be done in a follow-up PR that might be a bit hefty, but without much 'content'. I think doing that separately might also make it a bit easier to see what relies on what.

Agree that it would be good to keep an eye on naming/how the stack looks